TranslationApp/app.config

```diff
@@ -10,14 +10,21 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
💭 Need to see if `<AutoGenerateBindingRedirects>` works with this project system. If so, it's much easier.

Reply:
I'll investigate, thanks for the pointer.

Reply:
it looks like it worked, thank you

Reply:
Nice 😎